utils/src/lib.rs

```diff
@@ -632,7 +634,7 @@ pub fn abi_to_solidity(contract_abi: &Abi, mut contract_name: &str) -> Result<St
     };
 
     // instantiate an array of all ABI Encoder v2 structs
-    let mut structs = HashSet::new();
+    let mut structs = BTreeSet::new();
```

Comment:
Do we do btreeset so that order is preserved?

Reply:
yes, so structs are also sorted

https://github.com/foundry-rs/foundry/blob/9fb4450a6d8f056f32b68a8153c6e7f0e0dbd45a/testdata/fixtures/SolidityGeneration/WithStructs.sol#L24-L27